projects/batfish/src/main/java/org/batfish/representation/iptables/IptablesChain.java

```diff
@@ -19,7 +19,7 @@
 
   private String _name;
 
-  private ChainPolicy _policy;
+  private ChainPolicy _policy = ChainPolicy.ACCEPT;
 
   private List<IptablesRule> _rules;
 
```

Comment:
Never mind, I was only running the maven tests, I see now that travis does a bit more.